datasync/chaindatafetcher/kafka/kafka.go

```diff
@@ -29,6 +29,7 @@ var logger = log.NewModuleLogger(log.ChainDataFetcher)
 const (
 	MsgIdxTotalSegments = iota
 	MsgIdxSegmentIdx
+	MsgIdxLength
```

Comment:
How about renaming these to `MsgHeader~~`?

Reply:
Reflected. e6e58c6